e2e/cypress/support/e2e.ts

```diff
@@ -9,6 +9,12 @@ compareSnapshotCommand({
   errorThreshold: 0,
 });
 
+Cypress.on("uncaught:exception", (err, runnable) => {
```

Comment:
We probably shouldn't be ignoring all exceptions. Is there specific one you were trying to avoid causing a faliure? if so, add it here.

Reply:
Right now we indiscriminately return false which means we're not ignoring any exceptions. I just wanted to keep it there in case we wanted to catch any known exceptions that could cause us trouble.